source/main.lua

```diff
@@ -180,6 +180,10 @@ end
 
 function love.load()
     if love.filesystem.isFused() then
+	
+		-- nullify the assert function for performance reasons
+		function assert() end
+	

```

Comment:
Can you explain what it does regarding performance? I never experienced a big impact in performance using assert to check statements and variables.